pkgs/by-name/sv/svndumpsanitizer/package.nix

```
@@ -0,0 +1,26 @@
+{lib, gccStdenv, fetchFromGitHub }:
+
+gccStdenv.mkDerivation rec {
+  pname = "svndumpsanitizer";
+  version = "2.0.7";
+  src = fetchFromGitHub {
+    owner = "dsuni";
+    repo = pname;
```

The name of the repo and the name of the package is not necessarily connected.

```suggestion
gccStdenv.mkDerivation {
  pname = "svndumpsanitizer";
  version = "2.0.7";
  src = fetchFromGitHub {
    owner = "dsuni";
    repo = "svndumpsanitizer";
```